tests/e2e/support/page/files/allFiles.ts

```diff
@@ -111,6 +121,39 @@ export class AllFilesPage {
     return downloads
   }
 
+  async downloadFilesBatchAction({
+    names,
+    folder
+  }: {
+    names: string[]
+    folder: string
+  }): Promise<Download[]> {
+    const { page } = this.actor
+    const startUrl = page.url()
+    const downloads = []
+    if (folder) {
+      await filesCta.navigateToFolder({ page: page, path: folder })
+    }
+    for (const name of names) {
+      await this.toggleFileOrFolderCheckbox(page, true, name)
+    }
+    const [download] = await Promise.all([
+      page.waitForEvent('download'),
+      this.batchActionDownloadLocator.click()
+    ])
+    downloads.push(download)
+    await page.goto(startUrl)
+    return downloads
+  }
+
+  async toggleFileOrFolderCheckbox(page, enable, path) {
```

Comment:
this way you can make optional param
```suggestion
  async toggleFileOrFolderCheckbox(page, path, checked=true) {
```